README.md

```diff
@@ -124,6 +128,25 @@ interface {
 }
 ```
 
+### Update the logging configuration
+
+> NOTE: You must provide at least one key/value pair as part of `config`
+
+```ts
+interface {
+  messageId: string;
+  command: "update_log_config";
+  config: {
+    enabled?: boolean;
+    level?: number;
+    transports?: Transport[];
```

Comment:
Can we expand the `Transport` type, not sure what it is here.

Reply:
I'm wondering if we should just drop it altogether (drop it from the `node-zwave-js` response before serialization)? As I mentioned in another PR, I think this is managed entirely internally. From what I can tell, transports are really just a stream writer that is created based on the other parameter values.

Reply:
yeah, in that case let's not expose more than we need right now for HA.